README.md

```diff
@@ -14,71 +16,252 @@ Takeaway Challenge
-  * Use the twilio-ruby gem to access the API
-  * Use the Gemfile to manage your gems
-  * Make sure that your Takeaway is thoroughly tested and that you use mocks and/or stubs, as necessary to not to send texts when your tests are run
-  * However, if your Takeaway is loaded into IRB and the order is placed, the text should actually be sent
-  * Note that you can only send texts in the same country as you have your account. I.e. if you have a UK account you can only send to UK numbers.
+## My own Story
 
-* Advanced! (have a go if you're feeling adventurous):
-  * Implement the ability to place orders via text message.
+```
+As a customer
+So that I can change my order if it is wrong
+I would to be able alter my order before I confirm
+```
+From the users perspective in IRB:
+```
+order = Order.new
+=> #<Order:0x00007f863783fda0 @menu={:fish=>5, :chips=>3, :salad=>2}, @cost=0, @items=[]> 
+order.update("1 fish, 3 salad")
+=> {:fish=>1, :salad=>3}
+order.cost
+=> 11
+order.update("2 fish, 0 salad")
+=> {:fish=>2}
+order.cost
+=> 10
+```
 
-* A free account on Twilio will only allow you to send texts to "verified" numbers. Use your mobile phone number, don't worry about the customer's mobile phone.
+## Domain model
 
-> :warning: **WARNING:** think twice before you push your **mobile number** or **Twilio API Key** to a public space like GitHub :eyes:
->
-> :key: Now is a great time to think about security and how you can keep your private information secret. You might want to explore environment variables.
+Objects | Messages
+--------|--------
+Menu| @list
+Order | @menu <br/>  @items <br/> @cost <br/> update(items) <br/> confirm
 
-* Finally submit a pull request before Monday at 9am with your solution or partial solution.  However much or little amount of code you wrote please please please submit a pull request before Monday at 9am
+## Class Diagram
 
+![image](./images/diagram4.png)
 
-In code review we'll be hoping to see:
+# Instructions
 
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
+## Install dependency
 
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance will make the challenge somewhat easier.  You should be the judge of how much challenge you want this weekend.
+- This program requires some gems to be installed. They can be installed via terminal in the main directory with:
 
-Notes on Test Coverage
-------------------
+```
+bundle
+```
+
+## SMS Integration
+
+- This program can be configured to work via text using Twilio.
+- You will need your own [Twilio account](https://www.twilio.com) to set up this up.
+- You will have to create the folowing file:
+```
+lib/send_sms.rb
+```
+- Add these contents to the file
+```
+require 'twilio-ruby'
+
+class Notification
```

Comment:
It might be a nice exercise to try and write tests for this file - how could we use mocking here to make sure our tests don't actually send text messages every time they're run?